queries/javascript/highlights.scm

```diff
@@ -103,7 +103,8 @@
 (null) @constant.builtin
 (comment) @comment
 (string) @string
-(regex) @string.special
+(regex) @punctuation.delimiter
+(regex_pattern) @string.regex
```

Comment:
We could add a  hl group for regexes...

Reply:
Wasn't the idea to use language injection for regex?

Reply:
indeed. but it hasn't arrived. but string should be good enough

Reply:
i guess string.regex is as good as regex :) maybe string.regex should just be regex though.

Reply:
i meant to say: string.regex is perfect for now